Gemfile

```diff
@@ -38,7 +38,6 @@ group :test do
   gem 'rspec-rails'
   gem 'rspec-its'
   gem 'factory_bot_rails'
-  gem 'database_rewinder'
```

Comment:
Not related to this PR. I found that we don't need this gem since `use_transactional_fixtures` does the same